bind/gencffi_cdef.go

```diff
@@ -5,10 +5,16 @@
 package bind
 
 import (
+	"go/types"
 	"strconv"
 	"strings"
 )
 
+func (g *cffiGen) genCdefStruct(s Struct) {
+	g.wrapper.Printf("typedef void* %s;\n", s.sym.cgoname)
```

Comment:
perhaps also generate some comment that says what this `void*` is? (ie with the Go name of the symbol being wrapped)